index.js

```diff
@@ -188,7 +164,7 @@ function sendNotification(endpoint, params) {
         options.headers['Authorization'] = 'Bearer ' + jwt;
         var key = 'p256ecdsa=' + urlBase64.encode(vapid.publicKey);
         if (options.headers['Crypto-Key']) {
-          options.headers['Crypto-Key'] += ',' + key;
+          options.headers['Crypto-Key'] += ';' + key;
```

Comment:
If I put in a comma with GCM, it actually broke the GCM API and discussing this with @beverloo he pointed out:

 > Semi-colon separates fields in a single header value, comma separates header values.

which I cant find in a spec anywhere for semicolon but regarding comma [from this spec](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2):

> Multiple message-header fields with the same field-name MAY be present in a message if and only if the entire field-value for that header field is defined as a comma-separated list [i.e., #(values)]. It MUST be possible to combine the multiple header fields into one "field-name: field-value" pair, without changing the semantics of the message, by appending each subsequent field-value to the first, each separated by a comma. The order in which header fields with the same field-name are received is therefore significant to the interpretation of the combined field value, and thus a proxy MUST NOT change the order of these field values when a message is forwarded.

This *might* have been one of the issues I was having with VAPID support on Chrome.